src/dropdown/dropdown.component.ts

```diff
@@ -377,7 +379,7 @@ export class Dropdown implements OnInit, AfterContentInit, AfterViewInit, OnDest
 	/**
 	 * Set to `true` if the dropdown is closed (not expanded).
 	 */
-	menuIsClosed = true;
+	@Input() menuIsClosed = true;
```

Comment:
We can change the following to something like this & update all instaces of `menuIsClosed` calls to use `isOpen`.

```suggestion
	@Input() isOpen = false;

	set menuIsClosed (isClosed: boolean) {
		this.isOpen = !isClosed;
		this.changeDetectorRef.markForCheck();
	}
```

Reply:
I still want to run markForCheck here so it doesn't break anything on release. Teams are/could be using ViewChild to update this variable

Reply:
A notice of deprecation should also improve dev experience if they are using it.